paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -160,3 +182,14 @@ internal class PrimaryButton @JvmOverloads constructor(
         data class FinishProcessing(val onComplete: () -> Unit) : State()
     }
 }
+
+@Composable
+private fun LabelUI(label: String) {
+    Text(
+        text = label,
+        textAlign = TextAlign.Center,
+        color = colorResource(R.color.stripe_paymentsheet_primary_button_text),
+        style = PaymentsTheme.typography.h5,
+        modifier = Modifier.padding(start = 4.dp, end = 4.dp, top = 4.dp, bottom = 5.dp)
```

Comment:
Where are these padding values coming from?

Reply:
they came from the xml I turned into compose.